plugins/kn-image/README.md

```diff
@@ -0,0 +1,220 @@
+# Kn service build and deploy plugin
+
+## Prequirement
+Before you begin this task, you should prepare a kubernetes cluster which has been installed tekton-pipeline and access it.
+### Confirm tekton-pipeline has been installed
+```
+> kubectl get pod -n tekton-pipelines
+NAME                                           READY   STATUS    RESTARTS   AGE
+tekton-pipelines-controller-6d8f8c4f66-6lcwf   1/1     Running   0          5d4h
+tekton-pipelines-webhook-5697f86c7c-d5c9r      1/1     Running   0          3d
+tekton-triggers-controller-5f58c5775f-zshvf    1/1     Running   0          5d4h
+tekton-triggers-webhook-74497cb8dd-h9qxd       1/1     Running   0          3d
+```
+Ensure all pods are `running`.
+
+## Create build task first
+Please create the build task first from https://github.com/tektoncd/catalog
+### Such as buildpacks build task:
+- Download yaml from https://github.com/tektoncd/catalog/blob/master/buildpacks/buildpacks-v3.yaml as buildpacks-v3.yaml
```

Comment:
Is there a non-version specific link you can use? This one includes v3 so will need to be updated in subsequent releases...

Reply:
No, there is only one buildpack: `buildpacks-v3`, there is no other buildpack versions.